Example/Tests/UserTests.swift

```diff
@@ -177,43 +176,42 @@ class UserTests: QuickSpec {
                 expect(user.tokens).toNot(beNil())
             }
 
-            it("Should emit set event") {
+            it("Should update tokens") {
                 let user = User(state: .loggedOut)
-                var newTokens: TokenData?
-                _ = user.didSetTokens.register { newTokens = $0.new }
+                expect(UserTokensKeychain().data().first).to(beNil())
                 _ = try? user.set(accessToken: "hehe", refreshToken: "hehe", idToken: "hehe")
-                expect(newTokens).toNot(beNil())
+                expect(UserTokensKeychain().data().first).toNot(beNil())
             }
 
-            it("Should not emit set event if set to same") {
+            it("Should not change tokens if set to same") {
                 let user = User(state: .loggedOut)
-                var newTokens: TokenData?
                 _ = try? user.set(accessToken: "hehe", refreshToken: "hehe", idToken: "hehe")
-                _ = user.didSetTokens.register { newTokens = $0.new }
+                let tokens = UserTokensKeychain().data().first
+                expect(tokens).toNot(beNil())
                 _ = try? user.set(accessToken: "hehe", refreshToken: "hehe", idToken: "hehe")
-                expect(newTokens).to(beNil())
+                expect(UserTokensKeychain().data().first).to(equal(tokens))
```

Comment:
fyi: i think you can also do `expect(tokens) == tokens` instead of using `.to(equal...`